unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs

```diff
@@ -597,6 +600,28 @@ public BaseComponent EntityComponentCreateOrUpdateFromUnity(string entityId, CLA
                     newComponent = EntityUUIDComponentUpdate(entity, type, model);
                 }
             }
+            else
+            {
+                if (!entity.components.ContainsKey(classId))
```

Comment:
Let's invert the if. Negated logic tends to be harder to maintain.

Reply:
This is legacy code copy and pasted from a similar function in the class. However, this function has a pending task of refactoring to use the same pipeline for the creation of components, I will change it with the refactor if that is ok with you!